README.md

```diff
@@ -188,6 +188,15 @@ $.Function :: Type
 
 Type comprising every Function value.
 
+#### `Integer`
+
+```haskell
+$.Integer :: Type
+```
+
+Type comprising every non-floating point [`ValidNumber`](#validnumber) value 
+between the range `-9007199254740992` and `9007199254740992`.
```

Comment:
How about this?

> Type comprising every integer in the range [[`Number.MIN_SAFE_INTEGER`][1] .. [`Number.MAX_SAFE_INTEGER`][2]].


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_SAFE_INTEGER
[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER